subsys/bluetooth/services/Kconfig.ddfs

```
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig BT_DDFS
+	bool "Direction and Distance Finding Service"
```

```suggestion
	bool "Direction and Distance Finding Service [EXPERIMENTAL]"
```

This isn't done automatically unfortunately

---

Fixed